File: app/src/main/java/de/garten/training/depotflow/data/api/dto/SyncResponseDto.java
package de.garten.training.depotflow.data.api.dto;

import java.util.List;

public class SyncResponseDto {
    public boolean success;
    public List<String> acceptedAggregateIds;
    public List<String> rejectedAggregateIds;
    public String serverTime;

    public SyncResponseDto() {
    }
}

File: app/src/main/java/de/garten/training/depotflow/data/api/dto/SyncResponseDto.kt
package de.garten.training.depotflow.data.api.dto

class SyncResponseDto {
    @JvmField
    var success: Boolean = false

    @JvmField
    var acceptedAggregateIds: List<String>? = null

    @JvmField
    var rejectedAggregateIds: List<String>? = null

    @JvmField
    var serverTime: String? = null
}
